Symfony/CS/Tests/Fixtures/Integration/empty_return,useless_return,whitespacy_lines,extra_empty_lines.test

```diff
@@ -0,0 +1,45 @@
+--TEST--
+Integration of fixers: empty_return,useless_return,whitespacy_lines,return,extra_empty_lines.
+--CONFIG--
+level=none
+fixers=empty_return,useless_return,whitespacy_lines,return,extra_empty_lines
+--INPUT--
+<?php
+class Test {
+
+    private static function a($a) {
+        if ($a) {
+            return 1;
+        }
+        /**/
+        ++$a;return ;
+    }
+
+    private static function b($a) {
+        if ($a) {
+            return;
+        }
+
+        return null;
+    }
+}
+
+--EXPECT--
```

Comment:
:+1:  
which is RTM, so let me switch those test around :)
(expecting tests to fail)